https://github.com/trusteddomainproject/OpenDMARC/issues/163

mdomsch

commit ab1bc01 removed conf_holdquarentinedmessages, but not completely.
A reference to it still exists which causes the build to fail.

--- a/opendmarc/opendmarc.c
+++ b/opendmarc/opendmarc.c
@@ -3591,7 +3591,6 @@
 		ret = SMFIS_CONTINUE;
 
 		if (conf->conf_rejectfail &&
-		    conf->conf_holdquarantinedmessages &&
 		    random() % 100 < pct)
 		{
 			snprintf(replybuf, sizeof replybuf,
